### ISSUE TITLE
Error with Integration_With_Slack

### ISSUE THREAD

**Issue by SupaHam** [JetBrains/youtrack-workflows#11]
When updating state of issue in project I get this error:
https://pastebin.com/vxC6Efg0

YouTrack version: Build 34480 Fri, 07 Jul 2017 13:24:07 GMT

**Comment by mariyadavydova** [comment]
@SupaHam 
This errors almost always means that the workflow has generated incorrectly in a workflow editor (this happens sometimes).
There are two ways to fix it:
* Download the latest version of the workflow from this repo and reattach it to your ptojects.
* If you use workflow editor, you can just check that everything is fine with this workflow on your instance (nothing is highlighted red) and force upload it to your server (i.e. click with right mouse button and select 'Force Upload' to make editor to update only this particular workflow).

**Comment by SupaHam** [comment]
Yeah a lot of stuff like `comments` is highlighted red. I didn't change anything in the workflow (other than URL) and the project, it had those errors from the moment I imported it.

**Comment by mariyadavydova** [comment]
@SupaHam Aha, you just needed to download the custom fields scheme (it comes with workflows when you connect to server). The error you've shown is harmless. If you force upload this workflow to your server now, it should work as expected.

**Comment by SupaHam** [comment]
Hi @mariyadavydova 

I've tried again, and I still get an [error](https://pastebin.com/HZqq9kP5).

**Comment by mariyadavydova** [comment]
The error is now different, but refers to the same problem: incorrect generation of the workflow from an editor. 
Would you mind to go to https://youtrack.jetbrains.com/newIssue?project=JT, create an issue and attach several things?
1. Your instance logs (or tell us the url of your instance, if you use InCloud YouTrack).
2. Exported slack workflow (to export workflow please go to Workflows administration page, select this workflow and click on export - you'll get a ZIP file, which I want to investigate).

**Comment by SupaHam** [comment]
@mariyadavydova There's so many log files. I presume you want me to zip the logs folder? If so, I am not comfortable sharing IP addresses and other target-able information. If you don't need the whole logs folder, could you please give me a file name?

**Comment by SupaHam** [comment]
I've decided to disable the `Post Resolved State to Slack` rule and it works fine.

I don't recall seeing this before, but when I add a project to the workflow, it says "requires setup". I go into `Post Priority Change to Slack` and offers to add a field `High` to priority field.

**Comment by SupaHam** [comment]
I've fixed my error, which surprisingly also tailors behaviour to what I want. I've modified the Workflow to remove the priority `becomes` check, thus requiring no dependency on a Priority state `High`.

Once I removed that last AND expression, uploaded, and tested. All messages go through slack just fine.

FWIW, I've also made a modification to replace the message output from fixed `"... High:"` to `issue.Priority`.

Sorry for taking up your time.
